everett/manager.py

```diff
@@ -46,6 +48,30 @@ def parse_bool(val):
     raise ValueError('%s is not a valid bool value' % val)
 
 
+def parse_env_file(envfile):
+    """Parse the content of an iterable of lines as .env
+
+    Return a dict of config variables.
+
+    >>> parse_env_file(['DUDE=Abides'])
+    {'DUDE': 'Abides'}
+
+    """
+    data = {}
+    for line in envfile:
+        line = line.strip()
+        if not line or line.startswith('#') or '=' not in line:
```

Comment:
I'm not sure about the "if there's no = in the line" part here. Wouldn't that suggest that the env file is malformed and therefore we should throw an exception?